src/msg/Message.cc

```diff
@@ -173,7 +173,7 @@ using namespace std;
 
 #define dout_subsys ceph_subsys_ms
 
-void Message::encode(uint64_t features, bool datacrc)
+void Message::encode(uint64_t features, bool datacrc, bool compression)
```

Comment:
we'll want to rebase this on top of the patch that changes this bool to a flags field.